openedx/core/djangoapps/content_tagging/migrations/0007_system_defined_org_2.py

```diff
@@ -0,0 +1,28 @@
+from django.db import migrations
+
+
+def mark_language_taxonomy_as_all_orgs(apps, _schema_editor):
+    """
+    Associates the system defined taxonomy Language (id=-1) to all orgs.
+    """
+    TaxonomyOrg = apps.get_model("content_tagging", "TaxonomyOrg")
+    TaxonomyOrg.objects.create(taxonomy_id=-1, org=None)
```

Comment:
Anyone running these migrations from what's in the `master` branch already has a `TaxonomyOrg` entry created from the fixture. And so after applying this migration, we now have 2 nearly identical entries:

```sql
mysql> select * from content_tagging_taxonomyorg
    -> ;
+----+----------+--------+-------------+
| id | rel_type | org_id | taxonomy_id |
+----+----------+--------+-------------+
| -1 | OWN      |   NULL |          -1 |
|  1 | OWN      |   NULL |          -1 |
+----+----------+--------+-------------+
2 rows in set (0.00 sec)
```

(This situation also prevents the rollback method `revert_mark_language_taxonomy_as_all_orgs` from running, because [`TaxonomyOrg.objects.get(taxonomy_id=-1, org=None)`](https://github.com/openedx/edx-platform/pull/33438/files#diff-db1c6e6ae6d077ce256b45a9f58b48af64b7704d35c082e7f33ad1866ea8e227R17) returns 2 rows, and so throws an exception.)

But if we change this line, then this migration applies and rolls back without issue:

```suggestion
    TaxonomyOrg.objects.update_or_create(taxonomy_id=-1, defaults={"org"=None})
```